src/screens/HomeScreen.js

```diff
@@ -239,6 +239,7 @@ HomeScreen.navigationOptions = ({ navigation, navigationOptions }) => {
           accessibilityHint="Zu den Einstellungen und Lesezeichen wechseln"
         >
           <Icon
+            size={normalize(44)}
```

Comment:
see above. size does not do anything here + is not what we want

Reply:
https://github.com/ikuseiGmbH/smart-village-app-app/pull/234#discussion_r651031564